### PR TITLE
Add structural card subcomponents

### PR DESCRIPTION
**Pull request by styczynski**
Added subcomponents for cards (in fact cards are visually displayed as one-side bordered boxes).
* title
* description
* image

image

### CODE REVIEW

File: frontend/src/components/universal/CardImage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import styled  from 'styled-components';
+
+import Logo, { LogoType } from 'components/Logo';
+
+interface WrapperProps {
```
Comment (mbalc) [inline frontend/src/components/universal/CardImage/index.tsx]: redundant code

File: frontend/src/components/universal/Card/index.tsx [hunk 2]
```diff
@@ -11,16 +15,52 @@ const Wrapper = styled.div`
   color: ${(props) => props.theme.colors.accent};
 `;
 
+const CardImageCell = styled.td`
+  padding-top: 0;
+`;
+
+const CardTitleCell = styled.td`
```
[the remark this answers is not in the document]
Comment (styczynski) [reply frontend/src/components/universal/Card/index.tsx]: This padding nonsensense was removed.

File: frontend/src/components/universal/Card/index.tsx [hunk 3]
```diff
@@ -11,16 +15,52 @@ const Wrapper = styled.div`
...
+    image?: string;
 }
 
 export default class Card extends React.Component<CardProps, undefined> {
     
     render() {
         return (
             <Wrapper>
-                {this.props.children}
+                <table>
```
[the remark this answers is not in the document]
Comment (styczynski) [reply frontend/src/components/universal/Card/index.tsx]: The table layout was changed to css flex styling.

File: frontend/src/components/universal/Card/index.tsx [hunk 4]
```diff
@@ -11,16 +15,52 @@ const Wrapper = styled.div`
...
+                            }
+                            <CardTitleCell>
+                                <CardTitle>
+                                    {this.props.title}
+                                </CardTitle>
+                            </CardTitleCell>
+                        </tr>
+                    </tbody>
+                </table>
+                {
```
[the remark this answers is not in the document]
Comment (styczynski) [reply frontend/src/components/universal/Card/index.tsx]: The description subcomponent is now incorporated in normal UI flow.

File: frontend/src/components/universal/CardTitle/index.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+const Wrapper = styled.div`
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
+  color: ${(props: WrapperProps) => props.theme.colors.accent};
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
+  font-size: ${(props: WrapperProps) => props.theme.fontSize[props.fontSize || 'XL']};
+`;
+
+export interface CardTitleProps {
+    children: string;
```
[the remark this answers is not in the document]
Comment (styczynski) [reply frontend/src/components/universal/CardTitle/index.tsx]: This was fixed.

File: frontend/src/components/universal/CardImage/index.tsx (hunk 1 above)
Comment (styczynski) [reply frontend/src/components/universal/CardImage/index.tsx]: The dead code was removed.

**Pull request merged**